$MirOS: ports/misc/mc/patches/patch-vfs_extfs_lslR_in,v 1.1 2008/09/17 20:06:59 tg Exp $
--- vfs/extfs/lslR.in.orig	Sun Jun 22 09:53:59 2003
+++ vfs/extfs/lslR.in	Fri Dec 31 17:44:06 2010
@@ -12,6 +12,8 @@ AWK=@AWK@
 
 mclslRfs_list () {
 case "$1" in
+  *.xz) MYCAT="xz -dc";;
+  *.lzma) MYCAT="lzma -dc";;
   *.bz2) MYCAT="bzip2 -dc";;
   *.gz)  MYCAT="gzip -dc";;
   *.z)   MYCAT="gzip -dc";;
